bin/GameIndex.dbf

```diff
@@ -15841,7 +15875,7 @@ Serial = SLES-54308
 Name   = Phantasy Star Universe
 Region = PAL-M3
 Compat = 5
-eeRoundMode = 0
+eeRoundMode = 2 // Fix for ennemies not moving.
```

Comment:
I assume this only affects battles, so `Fixes enemies not moving during battles.`